src/fundus/scraping/pipeline.py

```diff
@@ -53,9 +169,44 @@ def crawl(
         restrict_sources_to: Optional[List[Type[URLSource]]] = None,
         error_handling: Literal["suppress", "catch", "raise"] = "suppress",
         only_complete: Union[bool, ExtractionFilter] = True,
-        batch_size: int = 10,
+        delay: Optional[Union[float, Delay]] = None,
+        url_filter: Optional[URLFilter] = None,
+        only_unique: bool = True,
     ) -> Iterator[Article]:
+        """Yields articles from initialized publishers
+
+        Args:
+            max_articles (Optional[int]): Number of articles to retrieve. If there are fewer articles
+                than max_articles the Iterator will stop before max_articles. If None, all retrievable
+                articles are returned. Defaults to None.
+            restrict_sources_to (Optional[List[Literal["rss", "sitemap", "news"]]]): Let's you restrict
+                sources defined in the publisher specs. If set, only articles from given source types
+                will be yielded.
+            error_handling (Literal["suppress", "catch", "raise"]): Define how to handle errors
```

Comment:
Its probably best to have a look at best practices for doc strings and rework this accordingly.

Reply:
Could you give a little bit more context about your comment? Especially: Whats the problem here? A best practice for what?

Reply:
In my opinion, the sentence "Let's you restrict   sources defined in the publisher specs."  is an example of 'personal tone': In my opinion, documentation should state the facts, not what these facts are useful for. But I am not sure if this particular opinion of mine is supported by the best practices for writing doc strings. Therefore, it is necessary to do some research on best practices for doc string writing style and evaluate my opinion properly. This might be done after this particular pr has taken place to allow timely progress on this pr.